bin/sl-meshctl.js

```diff
@@ -144,6 +144,10 @@ function cmdStatus(instance) {
     fmt(1, 'port', '%s', rsp.port);
     fmt(1, 'base', '%s', rsp.base);
 
+    if (rsp.version) {
```

Comment:
I think a JSON dump would be perfectly fine in this case.. it's more of a debug/diagnostics command anyway..